### DISCUSSION TITLE
Light map style

### DISCUSSION THREAD

**Discussion by kzemlyak** [maplibre/maplibre-gl-js#1848] [Q&A]
Can I use the mapbox://styles/mapbox/light-v10 style for maplibre? I want the map to always be grey, like in this example https://docs.mapbox.com/mapbox-gl-js/example/custom-style-layer/

**Comment by HarelM** [comment]
I think you should ask mapbox or read their license.
In terms of protocol, maplibre doesn't support it or of the box but there's a plugin someone made that allows its usage.

**Comment by walkerke** [comment]
I'd recommend the Carto Positron style (`https://basemaps.cartocdn.com/gl/positron-gl-style/style.json`) or a "light" variant of a MapTiler style, e.g. `https://api.maptiler.com/maps/basic-v2-light/style.json`.
